todo_app/app.py

```diff
@@ -9,42 +9,58 @@
 import requests
 from todo_app.data.user import User
 from todo_app.data.roleChecker import ReadWriteNeeded
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
 
 
 def create_app():
     app = Flask(__name__)
     app.config.from_object(Config())
     mongoDB = MongoDB()
     to_do_list_local = to_do_list(mongoDB)
+    app.logger.setLevel(app.config['LOG_LEVEL'])
+
+    if app.config['LOGGLY_TOKEN'] is not None:
+        handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app')
+        handler.setFormatter(Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        app.logger.addHandler(handler)
 
     login_manager = LoginManager()
 
+
     @login_manager.unauthorized_handler
     def unauthenticated():
         url = f'https://github.com/login/oauth/authorize?client_id={os.getenv("GIT_HUB_CLIENT_ID")}'
         return redirect(url)
 
     @login_manager.user_loader
     def load_user(user_id):
-        return User(user_id)
+        user = User(user_id)
+        app.logger.info("User is " + user.role)
```

Comment:
This looks reasonable, but you'll find it executes on every user request - with even a handful of users accessing the system you may quickly find that you just have tons of logs reporting what roles are being seen, so switching this to `debug` level feels sensible